optuna/study.py

```diff
@@ -304,6 +304,10 @@ def objective(trial):
                 Flag to show progress bars or not. To disable progress bar, set this ``False``.
                 Currently, progress bar is experimental feature and disabled
                 when ``n_jobs`` :math:`\\ne 1`.
+
+        Raises:
+            RuntimeError:
+                If nested invocation of this method occurred.
```

Comment:
Sorry for the late review but could you just fix the tense to be consistent with e.g. https://github.com/optuna/optuna/blob/master/optuna/storages/_base.py#L105 ?
```suggestion
                If nested invocation of this method occurs.
```

Reply:
Sure!